code/clue.js

```diff
@@ -1,59 +1,454 @@
+  },
+  {
+    firstName: 'Josephine',
+    lastName: 'White',
+    color: 'white',
+    description: 'Avid reader',
+    age: 45,
+    image: './assets/white.png',
+    occupation: 'Chef',
+    favourite: 'axe',
+  },
+];
 
-const rope = {
-  name: 'rope',
-  weight: 10
-}
+const weapons = [
+  {
+    name: 'rope',
+    weight: 10,
+    id: 'rope',
+    favourite: 'patio',
+  },
+  {
+    name: 'knife',
+    weight: 25,
+    id: 'knife',
+    favourite: 'kitchen',
+  },
+  {
+    name: 'candlestick',
+    weight: 105,
+    id: 'candlestick',
+    favourite: 'dining room',
+  },
+  {
+    name: 'dumbbell',
+    weight: 150,
+    id: 'dumbbell',
+    favourite: 'spa',
+  },
+  {
+    name: 'poison',
+    weight: 5,
+    id: 'poison',
+    favourite: 'conservatory',
+  },
+  {
+    name: 'axe',
+    weight: 85,
+    id: 'axe',
+    favourite: 'observatory',
+  },
+  {
+    name: 'bat',
+    weight: 30,
+    id: 'bat',
+    favourite: 'billiard room',
+  },
+  {
+    name: 'trophy',
+    weight: 20,
+    id: 'trophy',
+    favourite: 'living room',
+  },
+  {
+    name: 'pistol',
+    weight: 35,
+    id: 'pistol',
+    favourite: 'study',
+  },
+];
 
-// THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
+const rooms = [
+  {
+    name: 'dining room',
+    id: 'dining room',
+  },
+  {
+    name: 'conservatory',
+    id: 'conservatory',
+  },
+  {
+    name: 'kitchen',
+    id: 'kitchen',
+  },
+  {
+    name: 'study',
+    id: 'study',
+  },
+  {
+    name: 'library',
+    id: 'library',
+  },
+  {
+    name: 'billiard room',
+    id: 'billiard room',
+  },
+  {
+    name: 'lounge',
+    id: 'lounge',
+  },
+  {
+    name: 'ballroom',
+    id: 'ballroom',
+  },
+  {
+    name: 'hall',
+    id: 'hall',
+  },
+  {
+    name: 'spa',
+    id: 'spa',
+  },
+  {
+    name: 'living room',
+    id: 'living room',
+  },
+  {
+    name: 'observatory',
+    id: 'observatory',
+  },
+  {
+    name: 'theater',
+    id: 'theater',
+  },
+  {
+    name: 'guest house',
+    id: 'guest house',
+  },
+  {
+    name: 'patio',
+    id: 'patio',
+  },
+];
 
-// NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
+const mystery = {
+  killer: '',
+  weapon: '',
+  room: '',
+};
 
-const suspects = [
-  mrGreen,
-  mrsWhite
-  // ...  and the rest
-]
+/*** FUNCTIONS ***/
+
+/* Called when the 'Let's Play' button is clicked, toggles the window, hides
+ the Reveal button and shuffles the suspects' favourite weapon and the weapons'
+ favourite room */
+const startGame = (windowId) => {
+  toggleWindow(windowId);
+  hideButton();
+  document.getElementsByTagName('BODY')[0].style.background = '#333';
+  shuffleFavourite(suspects, weapons);
+  shuffleFavourite(weapons, rooms);
+};
+
+/* The function is invoked when the 'killer card' deck is clicked. It checks that
+ a killer hasn't already been picked, then toggles the loader animation. After 
+ 3 seconds a killer is randomly selected and added to the mystery object and 
+ then displayed  */
```

Comment:
I like the way you have a comment to explain how the function works. This makes me easier to understand your code. Good learning point.